src/file.ts

```diff
@@ -3816,7 +3836,7 @@ class File extends ServiceObject<File> {
  * that a callback is omitted.
  */
 promisifyAll(File, {
-  exclude: ['publicUrl', 'request', 'setEncryptionKey'],
+  exclude: ['publicUrl', 'request', 'save', 'setEncryptionKey'],
```

Comment:
is this considered a breaking change?

Reply:
It is not. Instead of `save` being promisified here at the end, it is promisified in the function itself.